### ISSUE TITLE
Default Values for IdentityUser

### ISSUE THREAD

**Issue by davidrevoledo** [aspnet/Identity#1986]
I've realized that the configuration for the default entity `IdentityUser` 
https://github.com/aspnet/Identity/blob/master/src/EF/IdentityUserContext.cs#L134 doesn't have any configuration for the `PhoneNumber` and other values.

I faced a few perfomance lacking with varchar(max) sizing.

Reference : https://sqlperformance.com/2017/06/sql-plan/performance-myths-oversizing-strings

I think it'd be a good idea use some standard as default to avoid users that don't care or don't know about it have to deal with those kind of throughput issues and every once can configure like they prefer (as it is possible now) but IMO is a quite dangerous letting this as default

![image](https://user-images.githubusercontent.com/8322252/46265030-2dda9280-c4f9-11e8-9649-881dec444d3b.png)

Acording this with `For full international support, you'd need a VARCHAR of 15 digits.`
https://en.wikipedia.org/wiki/E.164
 
Thanks.

**Comment by davidrevoledo** [comment]
any news ?

**Comment by KevMorelli** [comment]
I ran a few tests and it seems that these length values could work without issues:
PhoneNumber => 15
PasswordHash => 84
SecurityStamp => 32
ConcurrencyStamp => 36

**Comment by davidrevoledo** [comment]
@KevMorelli Yeah I'm aware of that.

**Comment by KevMorelli** [comment]
I will check the other tables before making a pull request with the fixed migration for the templates because there are many fields with maximum length.

**Comment by davidrevoledo** [comment]
@KevMorelli I think we should wait until some one of the team answer this Issue, the PR will be a quite complicated as will be a breaking change.

**Comment by ajcvickers** [comment]
@HaoK @blowdart I think there is value here, and for 3.0 we should probably bite the bullet and do schema cleanup, but clearly not before 3.0 given where we currently are. Suggest we triage this to 3.0.

**Comment by davidrevoledo** [comment]
@ajcvickers yes I think the same if not it will force everyonce to run a migration.
